app/models/user.rb

```diff
@@ -5,14 +5,14 @@ class User < ActiveRecord::Base
 
   PERMITTED_ATTRS = [:bio, :email, :handle, :hide_email, :location, :password, :website].freeze
 
-  has_many :rubygems, through: :ownerships
+  has_many :rubygems, through: :ownerships, dependent: :destroy
```

Comment:
It does not delete `rubygem`, it deletes the `ownerships`. Not saying this is a credible source, however: http://makandracards.com/makandra/32175-don-t-forget-automatically-remove-join-records-on-has_many-through-associations

I have added it to `ownership` line cause it does the same thing.. I guess it is more readable.